### DISCUSSION TITLE
Flaskit cart MD compatibility

### DISCUSSION THREAD

**Discussion by mvecinorosado** [sanni/cartreader#65] [Ideas & Suggestions]
Hi Sanni,

I recently saw some cartridges in Aliexpress that are a copy of the Flashkit cart PCB design and I would like to know if the cartreader could reprogram these cartridges because they look an interesting alternative to the cheap and bad quality bootlegs.

A link: https://www.aliexpress.com/i/1005001468335612.html

Thanks in advance!

**Comment by sanni** [comment]
Currently the Mega Drive code only supports writing MX29F1610 flashroms but if you compare both the datasheet of the MX29F1610 and the M29W640 used on the flashkit PCB and make the needed changes to the code it should be possible.
